lib/patches/tenant_run_concern.rb

```diff
@@ -1,7 +1,7 @@
 module Patches
   module TenantRunConcern
     def run(tenant_name, path = nil)
-      Apartment::Tenant.switch(tenant_name)
+      Apartment::Tenant.switch!(tenant_name)
```

Comment:
What about:

```
Apartment::Tenant.switch(tenant_name) do
  Patches::Runner.new(path).perform
end
```

As recommended by https://github.com/influitive/apartment#switching-tenants

I guess it's so that you don't end up staying switched to the last tenant at the end of the run.

It looks like the change was introduced in apartments v1.0.0:
https://github.com/influitive/apartment/blob/development/HISTORY.md#100

Reply:
Thanks @schlick. Good catch. I thought the block was introduced in v2.